JotunnLib/Managers/SynchronizationManager.cs

```diff
@@ -442,6 +442,9 @@ private void FejdStartup_Awake(On.FejdStartup.orig_Awake orig, FejdStartup self)
             // Harmony patch BepInEx to ensure locked values are not overwritten
             Harmony.CreateAndPatchAll(typeof(ConfigEntryBase_SetSerializedValue));
             Harmony.CreateAndPatchAll(typeof(ConfigEntryBase_GetSerializedValue));
+#if DEBUG
+            Harmony.CreateAndPatchAll(typeof(Debug_isDebugBuild));
```

Comment:
Why is this in the SyncManager? Please add to DebugUtils. That class is already guarded by the #Debug pragma